src/client/js/partials/breaches.js

```diff
@@ -68,6 +70,13 @@ function handleEvent (e) {
       state.emailCount = e.detail.newEmailCount
       renderZeroState()
       break
+    case e.type === 'toggle':
```

Comment:
The way the switch statement here is doing `switch (true)` and then matching on different kinds of things (some use the element name, some filter on the type) makes me a bit uncomfortable, I feel like it could lead to confusing bug. But, since we're not keeping this long-term, I felt like staying consistent with having a single handler here was more important.